### PR TITLE
Facebook-project-added

### PR DESCRIPTION
**Pull request by Lakshay-Pruthi**
I have added Facebook Sign-up page clone guided project.

### CODE REVIEW

File: facebook-SignUp-page-clone/spec.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Facebook Sign-up page project
+tags: ['HTML','CSS', 'Tailwind CSS']
+starter-files: https://github.com/Lakshay-Pruthi/facebook-clone-html-playground-starter
+type: frontend
+level: easy
+cover-image: https://github.com/Lakshay-Pruthi/facebook-clone-html-playground-starter/blob/master/Project-Design/Screenshot%202022-10-05%20175946.png
```
Comment (Pranavtechie) [inline facebook-SignUp-page-clone/spec.yml:6]: Need to add the direct image link : raw.githubusercontent. ...

File: facebook-SignUp-page-clone/spec.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: Facebook Sign-up page project
```
Comment (Pranavtechie) [inline facebook-SignUp-page-clone/spec.yml:1]: Can you change the folder name to only include lowercase letters. 

You may have to do a dummy commit, as git doesn't register changes if the case of the characters is changed. 
So make changes to the folder name commit changes and then make a change again (make sure that all the characters are lowercase now) and then commit again

File: facebook-SignUp-page-clone/step0/instructions.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+In this step you have to set the background color #f0f8fb
+
+ -Use "bg-[color-name]" class to achieve this goal
+
+ -Use the <img> tag and add the logo-image link
+
+Link for the logo image :
+```text
+"https://static.xx.fbcdn.net/rsrc.php/y8/r/dF5SId3UHWd.svg"
```
Comment (Pranavtechie) [inline facebook-SignUp-page-clone/step0/instructions.md:9]: Can you download this image and create an `/assets` folder in the boilerplate repo and add it there, it is recommended to have all static assets locally in the playground itself.

**Review by Pranavtechie — CHANGES_REQUESTED** [review]
You Need to rename all the instructionsmd file to `Instructions.md` (Capital Letter I) 

As mentioned earlier, you might also have to do a dummy commit here to change the character case of the file.

**Comment by Lakshay-Pruthi** [comment]
@Pranavtechie , I have made all the changes. Please review it.

**Comment by Pranavtechie** [comment]
Merging these commits from here https://github.com/codedamn/projects/pull/225

**Comment by Lakshay-Pruthi** [comment]
@Pranavtechie 👍🏻

**Pull request closed without merging**